modules/olm-operatorgroups-limitations.adoc

```diff
@@ -0,0 +1,11 @@
+[id="olm-operatorgroups-limitations"]
+= Limitations for multi-tenant Operator management
+
+OpenShift Container Platform provides limited support for installing different variations of an Operator on a cluster at the same time. Operators are control plane extensions. Since all tenants, or namespaces, share the same control plane of a cluster, tenants in a multi-tenant environment also have to share Operators
```

Comment:
Avoid using "have to": ...tenants in a multi-tenant environment also share Operators.